api/resources/translations/messages-en.properties

```diff
@@ -686,6 +687,7 @@ export.feedback.description = Download a log of detected errors and user feedbac
 export.messages.description = Download all messages that have ever been sent or received in CSV format.
 export.people.description = Download all contacts registered in the system in JSON format.
 export.reports.description = Download a summary of all the reports that have ever been submitted in CSV format.
+export.user-devices.description = Download a summary of the versions of the different pieces of software the CHT uses from users' devices in CSV format.
```

Comment:
What do you mean by `pieces of software the CHT uses`?

Reply:
It represents the version of the user's browser, Android OS, and of the CHT instance. I'm not 100% satisfied with this wording, you got an idea of how to express this more clearly?

cc @mrjones-plip @garethbowen if you have an alternative wording to suggest

Reply:
I think I get what you mean, but I'm not sure how to best say it in proper English; adding what those software are can help explain:
 
`Download a summary of software versions related to CHT (browser, CHT instance and OS) running in the user's devices. The summary is exported in CSV format.`

Once we have the message defined, I'll get the translation in Spanish and Indonesian, Ben can help with the Swahili and Sugat with the Nepali 😎 (our diversity feels powerful)

Reply:
@m5r what do you think about this suggestion?

Reply:
Oops sorry for missing it, it's a good one! If @jkuester agrees too, we should go ahead and get it translated  
I'll take care of the French one :smile:

Reply:
These are some translation you can use:

> User devices

- Spanish:
  - Dispositivos de usuario
- Indonesia:
  - Perangkat pengguna
- Swahili:
  - Vifaa vya mtumiaji

> Download a summary of software versions related to CHT (browser, CHT instance and OS) running in the user's devices. The summary is exported in CSV format.

- Spanish:
  - Descargue un resumen de las versiones de software relacionadas con CHT (navegador, instancia de CHT y sistema operativo) que se ejecutan en los dispositivos del usuario. El resumen se exporta en formato CSV.
- Indonesia:
  - Unduh ringkasan versi perangkat lunak terkait CHT (browser, instansi CHT, dan OS) yang berjalan di perangkat pengguna. Ringkasan tersebut diekspor dalam format CSV.
- Swahili:
  - Pakua muhtasari wa matoleo ya programu yanayohusiana na CHT (kivinjari, mfano wa CHT na tofauti za mifumo) unaoendeshwa kwenye vifaa vya mtumiaji. Muhtasari huo unahamishwa katika umbizo wa CSV.



Swahili by @Benmuiruri, thanks!

Reply:
Thanks a lot! :raised_hands: